xbrowse_server/phenotips/utilities.py

```diff
@@ -0,0 +1,121 @@
+import os
+import logging
+from django.conf import settings
+import json
+import urllib2
+import base64
+import requests
+logger = logging.getLogger(__name__)
+import sys
+
+#create a patient record in phenotips
+#by convention username and password are project_id,project_idproject_id
+#authentication is protected by access to machine/localhost
+def create_patient_record(individual_id,project_id,patient_details=None):
+  '''make a patient record'''
+  uri = settings.PHENOPTIPS_HOST_NAME + '/bin/PhenoTips/OpenPatientRecord?create=true&eid=' + individual_id
+  if patient_details is not None:
+    uri += '&gender='+patient_details['gender']
+  uname=project_id
+  pwd=project_id+project_id
```

Comment:
-Switched to "get_uname_pwd_for_project", this was an old iteration of code that was missed.
-Regarding merging the two types of user creation methods: Maybe the naming was ambiguous here. User creation happens ONLY in method "add_new_user_to_phenotips". While method "add_read_only_collaborator_phenotips_patient" is for associating an existing phenotips username to a patient, with ONLY read-only capabilities. The 2 underlying API calls and purpose of methods are vastly different, so merging is probably not a good idea and not reduce any code. Does that clarify things? If you still feel strongly about this, if so I could rename these 2 functions around to help reduce any ambiguity.